src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -1625,11 +1624,25 @@ int count_effective_slots(ClassAdListDoesNotDeleteAds& startdAds, ExprTree* cons
 }
 
 
+static int 
+CountMatches(std::vector<ClassAd *> ads, classad::ExprTree* constraint) {
```

Comment:
Shouldn't this be pass-by-reference for the ads vector?

Reply:
Yes.